backend/dataall/modules/metadata_forms/api/mutations.py

```diff
@@ -18,3 +24,36 @@
     resolver=delete_metadata_form,
     test_scope='MetadataForm',
 )
+
+createMetadataFormField = gql.MutationField(
+    name='createMetadataFormFields',
+    args=[
+        gql.Argument(name='formUri', type=gql.NonNullableType(gql.String)),
+        gql.Argument(name='input', type=gql.ArrayType(gql.Ref('NewMetadataFormFieldInput'))),
+    ],
+    type=gql.ArrayType(gql.Ref('MetadataFormField')),
+    resolver=create_metadata_form_fields,
+    test_scope='MetadataForm',
+)
```

Comment:
Well, It was at some point. I want to keep it for now, because I'm not sure I won't need it at all. I will remove it at the end of this stage of work, if there is no use. 
E.g. we may need it for CLI? ;)